Ryujinx/Ui/ApplicationLibrary.cs

```diff
@@ -346,21 +348,22 @@ private static IFileSystem GetControlFs(PartitionFileSystem Pfs)
             Nca controlNca = null;
 
             // Add keys to keyset if needed
-            foreach (DirectoryEntry ticketEntry in Pfs.EnumerateEntries("*.tik"))
+            foreach (DirectoryEntryEx ticketEntry in Pfs.EnumerateEntries("/", "*.tik"))
             {
-                Ticket ticket = new Ticket(Pfs.OpenFile(ticketEntry.FullPath, OpenMode.Read).AsStream());
+                Result rc = Pfs.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```

Comment:
```suggestion
                Result resultCode = Pfs.OpenFile(out IFile ticketFile, ticketEntry.FullPath, OpenMode.Read);
```
and add an empty line after.